src/components/_popovers.scss

```
@@ -32,6 +33,8 @@
     text-align: center;
     top: -6px;
     transform: translateX(-50%) translateY(-100%);
+    transition: opacity 0.2s ease-out, visibility 0s linear 0.2s;
```

Same as above, also you should remove the line with `@include transition(opacity);` as it is now overridden.
```suggestion
    transition: opacity 235ms ease-in-out, visibility 0s linear 0.2s;
```